docs/sections/user_guide/yaml/components/make_hgrid.rst

```diff
@@ -0,0 +1,252 @@
+
+nratio:
+"""""""
+
+Specify the refinement ratio when calculating cell length and area of supergrid.
+
+simple_dx:
+""""""""""
+
+Specify the uniform cell length in x-direction for simple cartesian grid.
+
+simple_dy:
+""""""""""
+
+Specify the uniform cell length in y-direction for simple cartesian grid.
+
+grid_name:
+""""""""""
+
+Specify the grid name.
+
+center:
+"""""""
+
+Specify the center location of grid.
+
+shift_fac:
+""""""""""
+
+Shift west by 180/shift_fac. Default value is 18.
+
+do_schmidt:
+"""""""""""
+
+Set to do Schmidt transformation to create astretched grid. When do_schmidt is set, the following must be set: --stretch_factor, --target_lon and --target_lat.
+
+do_cube_transform:
+""""""""""""""""""
+
+Set to re-orient the rotated cubed sphere so that tile 6 has 'north' facing upward. When do_cube_transform is set, the following must be set: --stretch_factor, --latget_lon, and --target_lat.  
+
+stretch_factor:
+"""""""""""""""
+
+Stretching factor for the grid.
+
+target_lon:
+"""""""""""
+
+Center longitude of the highest resolution tile.
+
+target_lat:
+"""""""""""
+
+Center latitude of the highest resolution tile.
+
+nest_grids:
+"""""""""""
+
+Set to create this # nested grids as well as the global grid.
+
+parent_tile:
+""""""""""""
+
+Specify the list of the parent tile number(s) of nest grid(s).
+
+refine_ratio:
+"""""""""""""
+
+Specify the list of refinement ratio(s) for nest grid(s).
+
+halo:
+"""""
+
+Halo size is to make sure the nest, including the halo region, is fully contained within a single parent (coarse) tile.
+
+istart_nest:
+""""""""""""
+
+Specify the list of starting i-direction index(es) of nest grid(s) in parent tile supergrid(Fortran index).
+
+iend_nest:
+""""""""""
+
+Specify the list of ending i-direction index(es) of nest grids in parent tile supergrid(Fortran index).
+
+jstart_nest:
+""""""""""""
+
+Specify the list of starting j-direction index(es) of nest grids in parent tile supergrid(Fortran index). 
+
+jend_nest:
+""""""""""
+
+Specify the list of ending j-direction index(es) of nest grids in parent tile supergrid(Fortran index).
+
+great_circle_algorithm:
+"""""""""""""""""""""""
+
+When specified, great_circle_algorithm will be used to compute grid cell area. 
+
+out_halo:
+"""""""""
+
+Extra halo size data to be written out.
+
+angular_midpoint:
+"""""""""""""""""
+
+If specified, when grid_type is from_file and the input is a NetCDF file, then the supergrid face midpoint coordinates are simply and independely (the lat independently from the lon) calculated as simple angular midpoints from the model grid coordiantes.
+
+non_length_angle:
+"""""""""""""""""
+
+When specified, will not output length(dx,dy) and angle (angle_dx, angle_dy).
+
+rotate_poly:
+""""""""""""
+
+Set to calculate polar polygon areas by calculating the area of a copy of the polygon, with the copy  being rotated far away from the pole.
+
+verbose:
```

Comment:
Is this an option that a user can add in the `config` block as a key? I didn't see it as an option in the example block provided, and wonder if it is does this add confusion to a user who will be using uwtools and see our verbose flag?  If it is an option for the `config` block should we add it in the YAML.

Reply:
Its a parameter for `make_hgrid`, so I put it as a key here, although its not required. Thats a valid concern, tho, I'm not exactly sure how to combine the two

Reply:
@maddenp-noaa I'm pondering the use case here, I'm not sure this flag is ever used by any of the apps. Would it be worth it to remove this as a YAML key and update the cli to take `--verbose` and driver itself to add `--verbose` to the runscript vars if used?

Reply:
I think that verbose behavior or `uw` and verbose behavior of `make_hgrid` should be kept orthogonal so that users can choose which (or both, or neither) they want. `uw --verbose` controls the former, so I'd assume that a `verbose` UW YAML key for `make_hgrid` would control the latter. If `make_hgrid` takes a verbose flag, I think they YAML entry would be the correct way to control that.